docs/metrics/accelerate.md

```diff
@@ -63,6 +63,15 @@ average(incident resolved timestamp - incident created timestamp)
 
 Pulse uses the maximum value over a period of time to display aggregate times to recover.
 
+<!--match-incident-deployment-start-->
+!!! note
+    We decided to avoid requiring a relationship between **incidents** and **deployments** or **changes** to simplify the data model reported.
+
+    As such, Pulse considers that the **deployment** for a system immediately before an **incident** for the same system is the one that caused that incident. The same deployment might be responsible for multiple incidents.
+
+    Pulse calculates the performance metrics per system and later aggregates the metrics by time interval when displaying them. This means that you need both deployments and incidents for Pulse to correctly map the system between the two types of events and for the metrics to work.
```

Comment:
I was hoping to make the case when users have events but still don't see data points explicit. something like:
```suggestion
    Pulse calculates the performance metrics per system and later aggregates the metrics by time interval when displaying them. This means that you need both deployments and incidents for Pulse to correctly map the system between the two types of events and for the metrics to work.
    Otherwise, even if you are receiving incident events, the charts will be empty.
```

Reply:
I added the sentence to clarify that the charts will be empty if the association between incidents and deployments through the system can't be made.